numba_dpex/core/kernel_interface/dispatcher.py

```diff
@@ -420,24 +446,51 @@ def __call__(self, *args, global_range=None, local_range=None):
             global_range, local_range, exec_queue.sycl_device
         )
 
-        # TODO: Enable caching of kernels, but do it using Numba's caching
-        # machinery
-
-        kernel = SpirvKernel(self.pyfunc, self.kernel_name)
-        kernel.compile(
-            arg_types=argtypes,
-            debug=self.debug_flags,
-            extra_compile_flags=self.compile_flags,
+        # TODO: Enable caching of kernels, but do it using LRU
```

Comment:
Can be removed?

Reply:
Yes, because this part has been executed in the next lines, please follow the `if-else` block after that.